<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DllRead</class>
 <widget class="QMainWindow" name="DllRead">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DllRead</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,2">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QToolBox" name="toolBox_2">
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;3ds&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>662</width>
         <height>740</height>
        </rect>
       </property>
       <attribute name="label">
        <string>频率波形-速度值</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <property name="leftMargin">
         <number>5</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>5</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QChartView" name="WavechartView">
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="SpeedchartView">
            <property name="contextMenuPolicy">
             <enum>Qt::ActionsContextMenu</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="font">
       <font>
        <family>3ds</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;3ds&quot;;</string>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>442</width>
         <height>666</height>
        </rect>
       </property>
       <attribute name="label">
        <string>初始化</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>401</width>
             <height>900</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>900</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>初始化</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>采样频率</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="freq">
                   <property name="maximum">
                    <number>10000</number>
                   </property>
                   <property name="value">
                    <number>4000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>采样数</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="freqn">
                   <property name="maximum">
                    <number>10000</number>
                   </property>
                   <property name="value">
                    <number>1000</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_11">
                 <item>
                  <widget class="QLabel" name="label_7">
                   <property name="frameShape">
                    <enum>QFrame::NoFrame</enum>
                   </property>
                   <property name="text">
                    <string>通道</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="channel">
                   <property name="minimum">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <number>8</number>
                   </property>
                   <property name="value">
                    <number>5</number>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_7">
              <property name="title">
               <string>采集卡测试</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
                 <item>
                  <widget class="QPushButton" name="TestButton">
                   <property name="text">
                    <string>句柄测试</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="state"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,1">
                 <item>
                  <widget class="QPushButton" name="cardset">
                   <property name="text">
                    <string>采集卡参数设置</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cardsettips"/>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1">
                 <item>
                  <widget class="QPushButton" name="cardquit">
                   <property name="text">
                    <string>退出采集卡</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLineEdit" name="cardquittips"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>信号选择</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_7" stretch="2">
               <item>
                <layout class="QGridLayout" name="gridLayout_2">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QRadioButton" name="physics">
                   <property name="text">
                    <string>采集卡</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QRadioButton" name="square">
                   <property name="text">
                    <string>方波</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QRadioButton" name="sin">
                   <property name="text">
                    <string>正弦波</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="QRadioButton" name="saw">
                   <property name="text">
                    <string>三角波</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>图形绘制</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
               <property name="spacing">
                <number>15</number>
               </property>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>刷新图形</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="timeperiod">
                   <property name="decimals">
                    <number>1</number>
                   </property>
                   <property name="maximum">
                    <double>20.000000000000000</double>
                   </property>
                   <property name="singleStep">
                    <double>0.100000000000000</double>
                   </property>
                   <property name="value">
                    <double>1.000000000000000</double>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_5">
                 <property name="spacing">
                  <number>10</number>
                 </property>
                 <item>
                  <widget class="QPushButton" name="StartAcquire">
                   <property name="text">
                    <string>开始采集</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="StopAcquire">
                   <property name="text">
                    <string>停止采集</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>状态提示</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_14">
               <item>
                <widget class="QLineEdit" name="tips">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>442</width>
         <height>666</height>
        </rect>
       </property>
       <attribute name="label">
        <string>信号</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="2,2,1">
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>速度仪表盘</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,1,1,1" columnstretch="2,1">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item row="0" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>上限</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="upper">
               <property name="maximum">
                <number>5000</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>4000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" rowspan="4">
            <widget class="QtRoundProgressBar" name="speedpanel" native="true">
             <property name="styleSheet">
              <string notr="true">background-color: rgba(255, 255, 255,0);</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>下限</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="lower">
               <property name="maximum">
                <number>100</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1">
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>转速比</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="ratio">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="value">
                <number>16</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="noTrans">
             <property name="text">
              <string>   无转换</string>
             </property>
             <property name="tristate">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>虚拟信号</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QtGaugeProgress" name="ampslide" native="true"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="text">
                <string>幅值</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QtGaugeProgress" name="freqslide" native="true"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="text">
                <string>频率</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'3ds'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;频率=零点个数/2&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;实际速度=频率/转速比/采样周期&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;转速=频率/16*60/采样周期&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;    =频率*3.75/采样周期&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>442</width>
         <height>666</height>
        </rect>
       </property>
       <attribute name="label">
        <string>测试页</string>
       </attribute>
       <widget class="QPushButton" name="convert">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>转换</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="waveconfig">
   <property name="icon">
    <iconset>
     <normaloff>../../icon/设 置.png</normaloff>../../icon/设 置.png</iconset>
   </property>
   <property name="text">
    <string>打开设置面板</string>
   </property>
  </action>
  <action name="speedconfig">
   <property name="icon">
    <iconset>
     <normaloff>../../icon/设 置.png</normaloff>../../icon/设 置.png</iconset>
   </property>
   <property name="text">
    <string>打开设置面板</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
  <customwidget>
   <class>QtGaugeProgress</class>
   <extends>QWidget</extends>
   <header>QtGaugeProgress.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtRoundProgressBar</class>
   <extends>QWidget</extends>
   <header>QtRoundProgressBar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
